spec/registry.md

```diff
@@ -538,6 +559,18 @@ function `:integer`:
 - `maximumSignificantDigits`
   - ([digit size option](#digit-size-options))
 
+If the _operand_ of the _expression_ is an implementation-defined type,
+such as the _resolved value_ of an _expression_ with a `:number` or `:integer` _annotation_,
+it can include option values.
+In general, these are included in the resolved option values of the _expression_,
+with _options_ on the _expression_ taking priority over any option values of the _operand_.
+Option values with the following names are however discarded if included in the _operand_:
```

Comment:
Does this mean "discarded" as in "not included in the resolved option values of the expression", or "discarded" as in "ignored completely"? I think it's the first one, but it could be misread as meaning the second one.

Reply:
Is there some observable difference between the two interpretations? These are the `:number` options that are not available on `:integer`, so I would presume that they would not have any impact on the selection or formatting either?

Reply:
Ah, I see -- I didn't catch that the list was listing options not available on `:integer`.